examples/lambda/docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM --platform=linux/amd64 public.ecr.aws/lambda/python:3.8
+
+WORKDIR /var/task
+
+# Copy the lambda and the requirements file
+COPY lambda/index.py .
+COPY lambda/aws_requirements.txt .
+
+# Install Python dependencies
+RUN pip install -r aws_requirements.txt -t .
```

Comment:
@malandis this seems like a fine way to load deps for this lambda but if you have other thoughts let us know

Reply:
LGTM